psycopg/psycopg/types/numeric.py

```diff
@@ -43,10 +52,6 @@ def dump(self, obj: Any) -> Buffer:
 
         return str(obj).encode()
```

Comment:
```suggestion
        return super().dump(obj)
```

Reply:
TBH I really don't like the need of an isinstance per int dump.

Now that we have Enum dumpers, I would like to try if it's possible to associate a pure int dumper to ints (one that just assumes that str(obj) returns the number representation) and associate a dumper converting to int only to enums.